pom.xml

```diff
@@ -63,7 +63,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.9</minimum>
+                                            <minimum>0.8</minimum>
```

Comment:
Changing this value isn't a great idea because it tells Travis how much of your code needs to be checked by your unit tests to pass. It looks like your previous commit wasn't building because some of your tests weren't diverse enough to test all parts of your code.

Here's a site that explains it better: https://mkyong.com/maven/maven-jacoco-code-coverage-example/

Reply:
Thanks for letting me know I'm gonna change that before I push it again